samples/subsys/usb/uac2_implicit_feedback/prj.conf

```diff
@@ -0,0 +1,16 @@
+CONFIG_I2S=y
+
+#USB related configs
+CONFIG_USB_DEVICE_STACK_NEXT=y
+CONFIG_USBD_AUDIO2_CLASS=y
+CONFIG_SAMPLE_USBD_PID=0x000F
+CONFIG_SAMPLE_USBD_PRODUCT="UAC2 implicit feedback sample"
+CONFIG_NET_BUF=y
+
+#LOG subsystem related configs
+CONFIG_LOG=y
+CONFIG_USBD_LOG_LEVEL_WRN=y
+CONFIG_UDC_DRIVER_LOG_LEVEL_WRN=y
+
+#Enable timer for asynchronous feedback
+CONFIG_NRFX_TIMER2=y
```

Comment:
Do we need to use a NRFX type timer here, or can a generic Zephyr timer be used?

Reply:
`feedback_nrf53.c` implementation uses timer and connects USB SOF and I2S FRAMESTART to it with DPPI. DPPI is so specific to Nordic that I don't really see how to achieve this without resorting to low-level hardware interface.

I don't know how to achieve proper feedback without relying on the low-level functionality. This however is something that application has to somehow figure out and I do provide a working sample implementation for nRF53.